dev/tasks/run-e2e

```diff
@@ -85,6 +85,14 @@ if [[ -z "${PREBUILT_TEST_BINARY:-}" ]]; then
   go test -c -o ${PREBUILT_TEST_BINARY} ./tests/e2e
 fi
 
+if [[ -n "${SKIP_TEST_APIGROUP:-}" ]]; then
```

Comment:
This is setup in the upper layer before calling runthe run-e2e https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/72e31ffad105c99a3689a7ef54e0f13666af46ef/scripts/github-actions/tests-e2e-fixtures#L45C45-L45C64 
Mind sharing the problem you found?

Reply:
It didn't seem to be skipping correctly.  I still don't see why not, but this works and before we were running e.g. bigtableinstance tests.